web/themes/custom/cloudy/src/components/alert/alert.twig

```diff
@@ -26,7 +26,7 @@
         } only %}
       {% endif %}
     </h5>
-    <div class="cloudy-alert__body">
+    <div class="alert__body">
       {{ content }}
     </div>
     {% if showTimestamp %}
```

Comment:
Interesting... I don't think this particularly timestamp meets our requirements for accessibility. I'm not sure how that got past the earlier review when this was being roles out. could you update the month to be the full month instead of the abbreviation?